README.md

```diff
@@ -75,101 +84,81 @@ To compile and build the `broadinstitute/firecloud-ui` docker image, run
-manually (see below). From the `automation/docker` directory:
+**Arguments:**
 
-```bash
-./run-tests.sh <# of nodes> <dev or qa> <ip of FiaB> automation $(cat ~/.vault-token)
-```
-
-Using 3 nodes should provide sufficient parallelization while fitting inside the recommended 4 GB
-docker memory setting when running on a developer workstation.
+* FireCloud location (required)
+	* One of `fiab`, `local`, `alpha`, `prod`, or an IP address. `fiab` will pull the IP from your `etc/hosts`.
+* `dev` or `qa`
+	* Environment of your FiaB. Defaults to `dev`.
+* Vault auth token
+	* Defaults to reading it from `~/.vault-token`.
+* Working directory
+	* Defaults to `$PWD`.
 
 ### Running directly (real Chrome)
 
-If you have a FiaB running and its IP is configured in your `/etc/hosts`, you can run tests locally
-and watch them execute in a browser. _IMPORTANT_: Be careful about interacting with your computer
-while running tests this way because stray keyboard and mouse interactions can interfere with test
-execution. It is recommended to use this only when writing/debugging specific tests, not running the
-entire test suite. 
+If you have a FiaB running and its IP is configured in your `/etc/hosts`, you can run tests locally and watch them execute in a browser. _IMPORTANT_: Be careful about interacting with your computer while running tests this way because stray keyboard and mouse interactions can interfere with test execution. It is recommended to use this only when writing/debugging specific tests, not running the entire test suite. 
 
 #### Set Up
 
-Before you run the tests for the first time, make sure you have chromedriver installed. On Mac OS,
-Homebrew (`brew`) is recommended:
+Before you run the tests for the first time, make sure you have chromedriver installed. On Mac OS, [Homebrew](https://brew.sh) is recommended:
 
 ```bash
 brew install chromedriver
 ```
 
-If your version of Chrome is 61 or later, check `chromedriver --version` to make sure you're up to
-at least 2.32. If not, use `brew upgrade chromedriver` to update.
+If your version of Chrome is 61 or later, check `chromedriver --version` to make sure you're up to at least 2.32. If not, use `brew upgrade chromedriver` to update.
  
 Also run the config render script. If you are planning on running the firecloud ui locally, add the local_ui param (it will set the baseUrl to "http://local.broadinstitute.org/". This will render the necessary `application.conf` and `firecloud-account.pem` for the tests. From the `automation` directory:
 
 ```bash
-./render-local-env.sh $PWD $(cat ~/.vault-token) <dev or qa> <local_ui>
+./render-local-env.sh [working dir] [vault token] [dev | qa] [local_ui]
 ```
 
+**Arguments:**
+
+* Working directory
+	* Defaults to `$PWD`.
+* Vault auth token
+	* Defaults to reading it from `~/.vault-token`.
+* `dev` or `qa`
+	* Environment of your FiaB. Defaults to `dev`.
+* Local UI
+	* Enter `local_ui` here to run against a local UI stack.
+
```

Comment:
add a comment that: if you specify this param, you must specify all params

Reply:
Ah, I'd thought that since they were positional, that was assumed.